apps/fis/fis-cos-api/demo-image-policy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImagePolicy
+metadata:
+  name: demo-fis-cos-api
+  annotations:
+    hmcts.github.com/prod-automated: disabled
+spec:
+  filterTags:
+    pattern: '^pr-165-[a-f0-9]+-(?P<ts>[0-9]+)'
+    extract: '$ts'
+  policy:
```

Comment:
remove duplication

Reply:
Can you give me a reference where this has been removed. As this piece of code is same across all the applications.